micro-rdk/src/esp32/tls.rs

```diff
@@ -261,6 +264,56 @@ impl Esp32TLSStream {
                             let mut fd: i32 = 0;
                             esp_idf_svc::sys::esp!(esp_tls_get_conn_sockfd(*tls_context, &mut fd))
                                 .map_err(|e| std::io::Error::new(std::io::ErrorKind::Other, e))?;
+
+                            // set socket keep alive properties as such
+                            // KEEPIDLE is set to 120 second (time before a first keepalive probe is sent)
+                            // KEEPINTVL, KEEPCNT are set to 60 and 4 respectively
+                            // Total time before an IDLE and DEAD connection is closed =  360s
+                            let var: i32 = 1;
+                            if lwip_setsockopt(
```

Comment:
what does this first option set? enables keepalive?

Reply:
yup